sympy/solvers/tests/test_inequalities.py

```diff
@@ -475,3 +476,28 @@ def test__pt():
     assert _pt(x, oo) == _pt(oo, x) == x + 1
     assert _pt(x, -oo) == _pt(-oo, x) == x - 1
     raises(ValueError, lambda: _pt(Dummy('i', infinite=True), S.One))
+
+
+def test_linear_programming():
+    A = Matrix([[0, 1, 2], [-1, 0, -3], [2, 1, 7]])
+    B = Matrix([3, -2, 5])
+    C = Matrix([[-1, -1, -5]])
+    D = Matrix([0])
+    optimum, argmax, argmax_dual = linear_programming(A, B, C, D)
+    assert optimum == -2
+    A = Matrix([[1, -1, 2], [-1, 2, -3], [2, 1, -7]])
```

Comment:
The tests should check what is returned for `argmax` and `argmax_dual` as well.

Add a blank line between the test cases because this function is quite dense.